cmd/explorer/main.go

```diff
@@ -125,11 +125,12 @@ var stages []stage = []stage{
 	{"RewriteEquals", "compile_stage_rewrite_equals"},
 	{"RewriteDynamicTerms", "compile_stage_rewrite_dynamic_terms"},
 	{"CheckRecursion", "compile_stage_check_recursion"},
-	{"CheckTypes", "compile_stage_check_types"}, // must be run after CheckRecursion
+	{"CheckTypes", "compile_stage_check_types"},
 	{"CheckUnsafeBuiltins", "compile_state_check_unsafe_builtins"},
 	{"CheckDeprecatedBuiltins", "compile_state_check_deprecated_builtins"},
 	{"BuildRuleIndices", "compile_stage_rebuild_indices"},
 	{"BuildComprehensionIndices", "compile_stage_rebuild_comprehension_indices"},
+	{"BuildRequiredCapabilities", "compile_stage_build_required_capabilities"},
```

Comment:
Not actually released yet, but it doesn't hurt, it'll just be ignored here until v0.59.0